From ac511fb0e8bf32c1a722ec2f544c64d7aaac7d0d Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Mon, 15 May 2017 11:39:09 +0000
Subject: [PATCH 0767/1000] MANTIS 0045822 MR-Component: VisionBox: PPR, Face;
 HRS: Iris

gmrdev.idl
V1.2
- added datatype DT_XB*, DT_USER_ID for biometrics




git-svn-id: svn://localhost/SelfServiceCommon/trunk@1201 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/gmrdev.idl | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/gmrdev.idl b/SelfServiceCommon/Massai/idl/Massai/gmrdev.idl
index 0dbfdd09..ba39ed21 100644
--- a/SelfServiceCommon/Massai/idl/Massai/gmrdev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/gmrdev.idl
@@ -24,7 +24,7 @@
 module gmrdev
 {
 
-   const string interface_version = "1.1";
+   const string interface_version = "1.2";
 
 /*
 using DeviceComponents::indicationListener and DeviceComponents::Generic from mdci.idl
@@ -147,8 +147,14 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
     DT_DISCRETIONARY_ISO, /* ISO track data with DISCRETIONARY Data truncation     */
     DT_FOID_JIS2,         /* JIS-2 track data with FOID Data truncation            */
     DT_PAYMENT_JIS2,      /* JIS-2 track data without truncation                   */
-    DT_DISCRETIONARY_JIS2 /* JIS-2 track data with DISCRETIONARY Data truncation   */
-  };
+    DT_DISCRETIONARY_JIS2,/* JIS-2 track data with DISCRETIONARY Data truncation   */
+    DT_XB01,              /* CUSS Biometric.xsd */  
+    DT_XB01_SETUP,        /* setup with CUSS Biometric.xsd */ 
+    DT_XB01_LEFT_EYE,     /* left eye */ 
+    DT_XB01_RIGHT_EYE,    /* right eye */ 
+    DT_XB01_FACE,         /* face */ 
+    DT_USER_ID            /* user identifaction */ 
+  };                      
 
   /**  The data status */
   enum dataStatus
-- 
2.41.0.windows.1

